### PR TITLE
Add Spanish translation for TypeScript recipee

### PR DESCRIPTION
**Pull request by kutyel**
I have already used the recipee and loved it! Now I would like to contribute by translating it! :wink:

### CODE REVIEW

File: es_ES/docs/recipes/typescript.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# TypeScript
```
Comment (forresst) [inline es_ES/docs/recipes/typescript.md:6]: Like other translated files, add this head before `# TypeScript`: 

``` markdown
___
**Nota del traductor**

Esta es la traducción del archivo [typescript.md](https://github.com/sindresorhus/ava/blob/master/typescript.md). Aquí hay un [enlace](https://github.com/sindresorhus/ava/compare/8e2f3dca177a4283ad882596d3c1425cabb998ef...master#diff-60cce07a584082115d230f2e3d571ad6) a las diferencias con la rama master de AVA (Si al hacer clic en el enlace no se encuentran modificaciones en el archivo `typescript.md`, será por que la traducción está actualizada).
___
```

File: es_ES/docs/recipes/typescript.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# TypeScript
+
+Traducciones: [Français](https://github.com/sindresorhus/ava-docs/blob/master/fr_FR/docs/recipes/typescript.md), [Русский](https://github.com/sindresorhus/ava-docs/blob/master/ru_RU/docs/recipes/typescript.md), [Español](https://github.com/sindresorhus/ava-docs/blob/master/es_ES/docs/recipes/typescript.md)
```
Comment (forresst) [inline es_ES/docs/recipes/typescript.md]: Put the Spanish link, first, to respect alphabetical order

**Comment by forresst** [comment]
@kutyel Great job ! (2 small points that you should change)

@AlbertoFuente Can you review the translation? Thanks

**Comment by kutyel** [comment]
@forresst true, had some issue while saving the file, added the translator's note in the first instance, now it is fixed. :sweat_smile: Thanks!

**Comment by kutyel** [comment]
Is this moving somewhere? I am very grateful to @SamVerschueren and @sindresorhus who assisted me in Twitter and I would like to do my bit by contributing this translation :wink:

**Comment by SamVerschueren** [comment]
@kutyel Happy I could help you out with it :). If you have any improvements, feel free to do a PR to the original recipe.

I guess @AlbertoFuente has to review the translation first. Not sure though.

**Comment by forresst** [comment]
@SamVerschueren  @kutyel Indeed, we are waiting for a review (@AlbertoFuente or another person who speaks Spanish)

**Comment by sindresorhus** [comment]
Can't wait forever. Merging.

Thanks @kutyel :)

**Pull request closed without merging**